docs/otlp/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,98 @@
+<!-- markdownlint-disable MD013 -->
+# Getting Started with OpenTelemetry Protocol (OTLP) in 5 Minutes - ASP.NET Core Application
+<!-- markdownlint-enable MD013 -->
+
+If you haven't already, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) and
+[Docker](https://www.docker.com/) on your computer.
+
+Install and run the [Standalone .NET Aspire
+dashboard](https://learn.microsoft.com/dotnet/aspire/fundamentals/dashboard/standalone)
+using Docker:
+
+```sh
+docker run --rm -it -p 18888:18888 -p 4317:18889 -p 4318:18890 -d --name aspire-dashboard mcr.microsoft.com/dotnet/aspire-dashboard:latest
+```
+
+> [!CAUTION]
+> The .NET Aspire dashboard is being used to view telemetry locally. It is a
+> developer tool and not meant for production usage.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.OpenTelemetryProtocol](../../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md),
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md),
+and
+[OpenTelemetry.Instrumentation.AspNetCore](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
+dotnet add package OpenTelemetry.Extensions.Hosting
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore
+```
+
+Update the `Program.cs` file with the code from [Program.cs](./Program.cs).
+
+Run the application (using `dotnet run`) and then browse to the .NET Aspire
+dashboard (eg `http://localhost:18888/`) to view your telemetry.
+
+Congratulations! You are now collecting traces using OpenTelemetry.
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/getting-started-jaeger would be nice to show 1-2 screenshots.